clusterloader2/testing/windows-tests/OWNERS

```diff
@@ -0,0 +1,3 @@
+approvers:
+- pjh
+- YangLu1031
```

Comment:
nit: if you could add newline at the end [I'm approving to not block you - please send a separate PR for it]

Reply:
@YangLu1031 ^^